packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2774,6 +2774,10 @@ class InputDecoration {
   /// of [InputDecoration] that changes the color of the label to the
   /// [InputDecoration.errorStyle] color or [ColorScheme.error].
   ///
+  /// When [label] is shown above the input field, it is scaled down
+  /// to [_kFinalLabelScale]. So, any given font size needs to be changed
```

Comment:
Please don't reference private symbols in doc comments

Reply:
Hey @goderbauer, I want to show the scale number, should I directly add 0.75 rather than using private variable. Or any other approach ?

Reply:
Putting 0.75 in the docs is alright, since otherwise folks won't be able to look up the private value without visiting the source code. :)